src/Compilers/Core/Portable/Operations/OperationInterfaces.xml

```diff
@@ -77,7 +77,7 @@
       </Comments>
     </Property>
   </Node>
-  <Node Name="ISwitchOperation" Base="IOperation">
+  <Node Name="ISwitchOperation" Base="IOperation" ChildrenOrder="Value,Cases">
```

Comment:
>ChildrenOrder [](start = 50, length = 13)

It seems like sometimes it's validated that all properties are present in ChildrenOrder, but properties like Locals are not included. What am I missing about the way the validation works?

Reply:
Children are only `IOperation` or arrays of `IOperation`.

---
In reply to: [315387892](https://github.com/dotnet/roslyn/pull/38056#discussion_r315387892) [](ancestors = 315387892)